go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/wintermute-cds/golevel7
```

Comment:
@dshills this commit broke the import path for your package when using go modules.

```go: finding github.com/dshills/golevel7 latest
go: github.com/diagnotes/trigger/hl7 imports
        github.com/dshills/golevel7: github.com/dshills/golevel7@v0.0.0-20190325155054-9d32fd2ecf6d: parsing go.mod:
        module declares its path as: github.com/wintermute-cds/golevel7
                but was required as: github.com/dshills/golevel7
```

I created a PR to fix this: https://github.com/dshills/golevel7/pull/2